BraintreeDropIn/Public/BTDropInRequest.h

```diff
@@ -71,6 +71,10 @@ typedef NS_ENUM(NSInteger, BTFormFieldSetting) {
 /// Defaults to false
 @property (nonatomic, assign) BOOL allowVaultCardOverride;
 
+/// Optional: Whether or not to vault the venmo upon tokenization, must be set to false when using a client token without a customer.
```

Comment:
If a merchant forgets to set this to `false`, and they use a client token without a customerID, won't this still crash since we're defaulting to `true`?

Reply:
It will result in an integration error and not a crash:

> This causes an error when attempting to use Venmo as a payment option as the SDK will error with "Failed to store Venmo Account in vault."

At that point, the integrator should note the error and fix the integration to set the flag to `false`. This approach was taken to avoid a breaking change.

Alternatively we could make a breaking change and default to `false` and make this part of the major version bump. In that case, tokenization will always work for initial integrations but merchants will need to opt-in to vaulting.